src/pages/components/map-related/MapLayers/BusToolTip.tsx

```diff
@@ -53,17 +57,20 @@ export function BusToolTip({ position, icon }: BusToolTipProps) {
           </header>
           <ul>
             <li>
-              {t('from')} :<span>{siriRide.gtfsRouteRouteLongName?.split('<->')[0]}</span>
+              {`${t('from')}: `}
+              <span>{siriRide.gtfsRouteRouteLongName?.split('<->')[0]}</span>
             </li>
             <li>
-              {t('destination')} :<span>{siriRide.gtfsRouteRouteLongName?.split('<->')[1]}</span>
+              {`${t('destination')}: `}
+              <span>{siriRide.gtfsRouteRouteLongName?.split('<->')[1]}</span>
             </li>
             <li>
-              {t('velocity')} :<span>{`${position.point?.velocity}  ${t('kmh')}`}</span>
+              {`${t('velocity')}: `}
+              <span>{`${position.point?.velocity}  ${t('kmh')}`}</span>
             </li>
 
             <li>
-              {t('sample_time')} :
+              {`${t('sample_time')}: `}
```

Comment:
why not
```suggestion
              {t('sample_time')}:
```
?

Reply:
The reason being the single space at the end of the string - doing it like you suggest will leave no space between the text and the data (unless we'll add an ugly `{' '}`) so I figured this is the cleanest approach to achieve the goal here